src/globals.c

```diff
@@ -652,6 +658,7 @@ void global_print_help(void)
 	print("   --debug                 Print extra information to help debugging problems\n");
 	print("   --no-progress           Don't print progress report\n");
 	print("   --wait-for-scripts      Wait for the post-update scripts to complete\n");
+	print("   --verbose               Enable verbosity for commands\n");
```

Comment:
can you please move this line between --quiet and --debug as you did in the documentation?
we are trying to group the options in the menus based on functionality, so --verbose belong between those two

Reply:
Sure thing Thanks